src/main/java/tigase/server/xmppclient/ClientConnectionManager.java

```diff
@@ -215,6 +215,12 @@ public void processPacket(final Packet packet) {
 			if (p.getStanzaFrom() != null) {
 				p.initVars(null, p.getStanzaTo());
 			}
+			if (serv.getAuthorisedUserJid().isPresent()) {
+				p.setServerAuthorisedStanzaFrom(serv.getAuthorisedUserJid().get());

```

Comment:
why not use `serv.getAuthorizedUserJid().ifPresent(jid -> p.setServerAuthorisedStanzaFrom(jid));`?

Reply:
Personal preference I guess? :-)
Probably more compact though...

Reply:
Not really, with `isPresent()` and `get()` you can have race condition and exception on get if something will change the value in between those checks, while `ifPresent()` is atomic.